src/main/java/uk/gov/companieshouse/web/accounts/Application.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.companieshouse.web.accounts;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Application {
```

Comment:
can we rename this to follow company accounts api? to `CompanyAccountsApplication`? Just to keep consistent